packages/arb-token-bridge-ui/src/pages/_app.tsx

```diff
@@ -50,10 +50,23 @@ Sentry.init({
     /^WebSocket connection failed for host: wss:\/\/relay.walletconnect.org$/i
   ],
   beforeSend: (event, hint) => {
+    if (!hint.originalException) {
+      return event
+    }
+
     if (isUserRejectedError(hint.originalException)) {
       return null
     }
 
+    const { code, message } = hint.originalException as {
+      code?: number
```

Comment:
https://docs.ethers.org/v5/api/utils/logger/#errors

i don't think `number` is correct, probably should use `ErrorCode`

<img width="1429" alt="image" src="https://github.com/user-attachments/assets/6565ab8d-51c6-4eef-b250-ad5d2f26ad55" />